pkg/webui/components/select/index.js

```diff
@@ -164,7 +164,7 @@ class Select extends React.PureComponent {
         className={cls}
         inputId={id}
         classNamePrefix="select"
-        value={getValue(translatedOptions, value)}
+        value={getValue(translatedOptions, value) || null}
```

Comment:
In theory, `''` can be a valid value. Maybe it's better to use null for `undefined` only, [since that can also not be set properly by formik](https://github.com/formium/formik/issues/2332).

Reply:
well, `''` can indeed be a valid value, but I dont think we should account for that. In any case where backend expects `''` for a certain field represented by `<Select />` a dummy/hardcoded value should be used and necessary conversion can be done in the submit function or validation schema.

Ill merge as it is now, if we decide to allow `''` in select it is pretty simple fix.